e2e/cleanup_test.go

```diff
@@ -19,12 +19,14 @@ const cleanupTest = "cleanup-test"
 
 func testCleanup() {
 	It("should create cleanup-test namespace", func() {
+		skipTestIfNeeded()
```

Comment:
I consider it is better to use just one `skipTestIfNeeded` to skip the entire test.
Can we wrap this test with new `It` and call `skipTestIfNeeded` in the first line of it?